http_logs/challenges/default.json

```diff
@@ -3,166 +3,14 @@
-        },
-        {
-          "operation": "scroll",
-          "warmup-iterations": 100,
-          "iterations": 200,
-          "#COMMENT": "Throughput is considered per request. So we issue one scroll request per second which will retrieve 25 pages",
-          "target-throughput": 1
-        },
-        {
-          "operation": "desc_sort_timestamp",
-          "warmup-iterations": 200,
-          "iterations": 100,
-          "target-throughput": 0.5
-        },
-        {
-          "operation": "asc_sort_timestamp",
-          "warmup-iterations": 200,
-          "iterations": 100,
-          "target-throughput": 0.5
-        },
-        {
-          "operation": "desc_sort_with_after_timestamp",
-          "warmup-iterations": 10,
-          "iterations": 100,
-          "target-throughput": 0.5
-        },
-        {
-          "operation": "asc_sort_with_after_timestamp",
-          "warmup-iterations": 10,
-          "iterations": 100,
-          "target-throughput": 0.5
-        },
-        {
-          "name": "force-merge-1-seg",
-          "operation": {
-            "operation-type": "force-merge",
-            "max-num-segments": 1,
-            "request-timeout": 7200
-          }
-        },
-        {
-          "name": "refresh-after-force-merge-1-seg",
-          "operation": "refresh"
-        },
-        {
-          "name": "wait-until-merges-1-seg-finish",
-          "operation": {
-            "operation-type": "index-stats",
-            "index": "_all",
-            "condition": {
-              "path": "_all.total.merges.current",
-              "expected-value": 0
-            },
-            "retry-until-success": true,
-            "include-in-reporting": false
-          }
-        },
-        {
-          "name": "desc-sort-timestamp-after-force-merge-1-seg",
-          "operation": "desc_sort_timestamp",
-          "warmup-iterations": 200,
-          "iterations": 100,
-          "target-throughput": 2
-        },
-        {
-          "name": "asc-sort-timestamp-after-force-merge-1-seg",
-          "operation": "asc_sort_timestamp",
-          "warmup-iterations": 200,
-          "iterations": 100,
-          "target-throughput": 2
-        },
-        {
-          "name": "desc-sort-with-after-timestamp-after-force-merge-1-seg",
-          "operation": "desc_sort_with_after_timestamp",
-          "warmup-iterations": 10,
-          "iterations": 100,
-          "target-throughput": 0.5
-        },
-        {
-          "name": "asc-sort-with-after-timestamp-after-force-merge-1-seg",
-          "operation": "asc_sort_with_after_timestamp",
-          "warmup-iterations": 10,
-          "iterations": 100,
-          "target-throughput": 0.5
-        }
+        {{ rally.collect(parts="common/default-schedule.json") }}
+      ]
+    },
+    {
+      "name": "append-runtime-script-grok",
+      "description": "Indexes the whole document corpus using scripts to extract fields.",
```

Comment:
Maybe we can add in the description that this relies on setting the track param `runtime_script_grok` to `true`.